indico/MaKaC/registration.py

```diff
@@ -4929,7 +4929,7 @@ class BilledItem(object):
 
     def __init__(self, caption, price, quantity, currency):
```

Comment:
The correct way to assign default price would be:

```python
def __init__(self, caption, quantity, currency, price=0.0):
```

Just make sure to update the order of arguments passed to `BilledItem()`.